### ISSUE TITLE
Upon connection, empty the files content for other tabs

### ISSUE THREAD

**Issue by zeachco** [lyuboraykov/vscode-share#1]
Cool project but not quite ready to share existing codebase.

**Comment by lyuboraykov** [comment]
@zeachco Thanks for the feedback! Yes, I did it as a university project, but I'm considering making it more robust.

Can you elaborate, what do you mean by `empty the files content for other tabs`?

**Comment by joeky888** [comment]
Nice plugin!

It seems that vscode will add this feature.

https://code.visualstudio.com/blogs/2017/11/15/live-share

**Comment by lyuboraykov** [comment]
Yes, true, most probably there wouldn't be a need for this extension when they release it.

But I'm glad, this means that the feature is useful and it will be widely available.

**Comment by zeachco** [comment]
Sorry for the late answer @lyuboraykov 
it's likely a race condition between two connected computer editing and saving the same file at almost the same time which cause one of the two clients to end up with an empty content (caused most of the time by the autosave setting but can be forced manually by abusing saves).
